modules/virt-launching-ibm-secure-execution-vm-ibm-z.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vm/virt-configuring-ibm-secure-execution-vms-ibm-z.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-launching-ibm-secure-execution-vm-ibm-z_{context}"]
+= Launching an {ibm-title} Secure Execution VM on {ibm-z-title} and {ibm-linuxone-title}
+
+Before launching an {ibm-name} Secure Execution VM on {ibm-z-name} and {ibm-linuxone-name}, you must add the `launchSecurity` parameter to the `secure-execution-vm.yaml` file . Otherwise, the VM does not boot correctly because it does not have access to the devices.
+
+.Procedure
+
+. Apply the following machine configuration:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: f41-se
+  name: f41-se
+spec:
+  runStrategy: Always
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: f41-se
+    spec:
+      domain:
+        launchSecurity: {}
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: rootfs
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 4Gi
+      terminationGracePeriodSeconds: 0
+      volumes:
+        - name: rootfs
+          dataVolume:
+            name: f41-se
```

Comment:
This is a very specific VM manifest; is it always these exact parameters? If not, please update it (using the description list format) so that it is clear which parameters need to be changed by the customer.